graphos/renderers/base.py

```diff
@@ -28,6 +28,13 @@ def __init__(self, data_source, html_id=None,
         self.encoder = encoder
         self.context_data = kwargs
 
+        self.chart_type = None
+        if kwargs:
+            self.chart_type = kwargs.get('chart_type')
```

Comment:
There shouldn't be any need to change base renderer to add more chart types of gchart.